### ISSUE TITLE
QUESTION: In StageB Finetune

### ISSUE THREAD

**Issue by 21Broccoli** [OpenDriveLab/UniVLA#74]
code：  
        action_pad = self.action_latent.expand(B, T, -1, -1)
        padded_patches = torch.cat([action_pad, dion_features], dim=2)
        action_pad_controllable = self.action_latent_controllable.expand(B, T, -1, -1)
        padded_patches = torch.cat([action_pad_controllable, padded_patches], dim=2)

Stage-1 :

  padded_patches = torch.cat([action_pad, dion_features], dim=2)
  position: [Action_Old(0-3), Image(4-259)]

Stage-2 :
  
  padded_patches = torch.cat([action_pad_controllable, action_pad, dion_features], dim=2)
  position: [Action_New(0-3), Action_Old(4-7), Image(8-263)]
                                                                                    ^^^^^^^^^^^^^ 
                                                                                      token shifting？！

**Comment by 21Broccoli** [comment]
In latent_action_model/genie/modules/lam.py line251~254
